recipes/poco/all/conanfile.py

```diff
@@ -139,6 +139,8 @@ def requirements(self):
                 self.options.enable_crypto or \
                 self.options.get_safe("enable_jwt", False):
             self.requires("openssl/1.1.1h")
+        if self.options.enable_data_odbc and self.settings.os != "Windows":
```

Comment:
Why this `!= "Windows`? Is the `enable_data_obdc` option not available in windows? Should we remove that option for that os then?

Reply:
odbc is a system lib on windows. `odbc` recipe should have been called `unixodbc`

Reply:
ok, thanks for the clarification. Yes, I see that the name could have been `unixobdc` but not a big issue 👍